classes/Task/Update/UpdateDatabase.php

```diff
@@ -53,7 +53,7 @@ public function run(): int
         try {
             if (!$this->container->getFileConfigurationStorage()->exists(UpgradeFileNames::SQL_TO_EXECUTE_LIST)) {
                 $this->warmUp();
-                $currentVersion = $this->container->getState()->getCurrentVersion();
```

Comment:
and again

Reply:
This file has one call to the state per method, storing it in a variable is not really important.